New docs for sunsetting a project

diemol opened the pull request:
There is a set of projects in the Sauce Labs org that have not been maintained in while and it would be ideal to understand if we want to sunset those projects or transfer them to a new owner.

To do that, we want to have first a guide to follow in these type of situations.

diemol committed:
New docs for sunsetting a project

diemol committed:
Adjusting netlify deploy

discombobulateme reviewed (approved):
I would recommend some slight changes: 

1. Having Signs to watch as part of the main description instead of part of the overview. Something like: how to identify or I know, its time for a project to be sunsetted? 
2. Instead of Overview: What to do/ Steps to sunset or something alike

Besides that, I think it's really great to have this resource!

diemol committed:
Merge branch 'dev' into sunsetting

diemol committed:
Update sunsetting.md

    Addressing PR comments
    
    - Rearranging sections
    - Being more concrete about approval chain
    - Time between announcement and archiving

diemol committed:
How to find dependencies

diemol committed:
Fixing linter
